<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>VEDA BANK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>361</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(147, 170, 157);</string>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="page_6"/>
   <widget class="QWidget" name="page_4">
    <widget class="QPushButton" name="pushButton_makeAcc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>Make a Account</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_deposit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>101</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>Deposit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_withdraw">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>101</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>Withdraw</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_allAcc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>101</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>all Accounts</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_exit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>101</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_spAcc">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>101</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>specific Account</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QDoubleSpinBox" name="spinBox_deposit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button_dconfirm">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>deposit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stackbutton_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>220</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>next</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Deposit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_dcancel">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>260</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Withdraw</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBox_withdraw">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button_wconfirm">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>withdraw</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stackbutton_5">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>220</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_wcancel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>260</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QPushButton" name="button_show">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>250</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>show</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="lineEdit_mn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ma">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBox_mb">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button_mconfirm">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>create</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stackbutton_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>260</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>next</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Make a Account</string>
     </property>
    </widget>
    <widget class="QPushButton" name="butto_mcancel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>48</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Account</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Balance</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="stackbutton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>440</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>stacked</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_namelogin">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
